CHANGELOG.md

```diff
@@ -32,10 +38,16 @@ All notable changes to this project will be documented in this file.
 * Clang-Format has been applied to the existing code-base of MoCOCrW and a `.clang-format`
   file has been included to format the code of of future PRs.
 * A foundational PKCS#11 HSM interface, based OpenSSL's ENGINE API, has been introduced
-  to MoCOCrW. Currently, the following functionality is supported:
+  to MoCOCrW. To build MoCOCrW with this API, follow the instruction in the [README](README.md).
+  Currently, the following functionality is supported:
     - Loading Public Keys
     - Loading Private Keys
     - Generating EC and RSA keypairs
+* Post Quantum Cryptography support has been added.
+  [Dilithium](https://www.pq-crystals.org/dilithium/)
+  is used to offer signing and verification functionality. To build MoCOCrW with dilithium support
+  please follow the instructions in the [README](README.md). Please note that the API for
+  dilithium will slightly change in the future once openssl officially supports dilithium.
```

Comment:
```suggestion
  Dilithium will slightly change in the future once openssl officially supports Dilithium.
```

Nit: Please capitalise Dilithium where appropriate throughout the diagram

Reply:
I guess you won't mind if we leave it as is :)